[2025-03-18T09:21:50.686+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-18T09:21:50.713+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: amazon_etl_dag.load_data scheduled__2025-03-18T08:00:00+00:00 [queued]>
[2025-03-18T09:21:50.721+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: amazon_etl_dag.load_data scheduled__2025-03-18T08:00:00+00:00 [queued]>
[2025-03-18T09:21:50.723+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-18T09:21:50.735+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data> on 2025-03-18 08:00:00+00:00
[2025-03-18T09:21:50.746+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'amazon_etl_dag', 'load_data', 'scheduled__2025-03-18T08:00:00+00:00', '--job-id', '265', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp2dgx6vs7']
[2025-03-18T09:21:50.749+0000] {standard_task_runner.py:91} INFO - Job 265: Subtask load_data
[2025-03-18T09:21:50.749+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=70240) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-18T09:21:50.750+0000] {standard_task_runner.py:63} INFO - Started process 70276 to run task
[2025-03-18T09:21:50.794+0000] {task_command.py:426} INFO - Running <TaskInstance: amazon_etl_dag.load_data scheduled__2025-03-18T08:00:00+00:00 [running]> on host 23e83e453497
[2025-03-18T09:21:50.881+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='amazon_etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-18T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-18T08:00:00+00:00'
[2025-03-18T09:21:50.882+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-18T09:21:50.954+0000] {load.py:64} INFO - Data successfully loaded into table 'amazon_products'
[2025-03-18T09:21:50.955+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-18T09:21:50.955+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-18T09:21:50.963+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=amazon_etl_dag, task_id=load_data, run_id=scheduled__2025-03-18T08:00:00+00:00, execution_date=20250318T080000, start_date=20250318T092150, end_date=20250318T092150
[2025-03-18T09:21:51.007+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-03-18T09:21:51.021+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-18T09:21:51.023+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
